superset-frontend/src/components/Datasource/CollectionTable.tsx

```diff
@@ -311,6 +320,25 @@ export default class CRUDCollection extends React.PureComponent<
     return <Icons.Sort onClick={this.sortColumn(col, 1)} />;
   }
 
+  renderTH(col: string, sortColumns: Array<string>) {
+    const tooltip = this.getTooltip(col);
+    return (
+      <th key={col} className="no-wrap">
```

Comment:
no-wrap is the real mvp I snuck into this PR

Reply:
fixing this monstruosity:
<img width="893" alt="Screen Shot 2023-06-12 at 4 47 23 PM" src="https://github.com/apache/superset/assets/487433/ee447513-981e-48f0-bc24-90b035404fba">